### ISSUE TITLE
2.0.0.backport1 is breaking sass-rails 3.2.6

### ISSUE THREAD

**Issue by PikachuEXE** [rails/sprockets-rails#98]
I just discover this gem has a backport version from [here](https://discussion.heroku.com/t/using-the-rails-4-asset-pipeline-in-rails-3-apps-for-faster-deploys/205)
So I was hoping to speed up asset compilation

But it throws an error on any sass file (the first one it encounters)
And even if the content of the file is empty, the error still exists
But when compile without `sprocket-rails` and with `sprocket` only, there is no error

Using
- `rails` `3.2.15`
- `sass-rails` `3.2.6`
- `sprocket` `2.2.2.backport2`

Backtrace:

```
undefined method `asset_paths' for #<#<Class:0x007fb069f021e0>:0x007fb067eec6d0>
  (in /Users/PikachuEXE/Projects/spacious-rails/app/assets/stylesheets/application-split2.scss)
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sass-rails-3.2.6/lib/sass/rails/template_handlers.rb:23:in `source_path'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sass-rails-3.2.6/lib/sass/rails/template_handlers.rb:84:in `sass_options'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sass-rails-3.2.6/lib/sass/rails/template_handlers.rb:106:in `evaluate'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/tilt-1.4.1/lib/tilt/template.rb:103:in `render'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/context.rb:197:in `block in evaluate'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/context.rb:194:in `each'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/context.rb:194:in `evaluate'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/processed_asset.rb:12:in `initialize'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:374:in `new'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:374:in `block in build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:395:in `circular_call_protection'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:373:in `build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:94:in `block in build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/caching.rb:58:in `cache_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:93:in `build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:287:in `find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:61:in `find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/bundled_asset.rb:16:in `initialize'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:377:in `new'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:377:in `build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:94:in `block in build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/caching.rb:58:in `cache_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:93:in `build_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/base.rb:287:in `find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/index.rb:61:in `find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:211:in `block in find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:257:in `benchmark'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:210:in `find_asset'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:119:in `block in compile'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:118:in `each'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/sprockets/manifest.rb:118:in `compile'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-rails-2.0.0.backport1/lib/sprockets/rails/task.rb:60:in `block (3 levels) in define'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-2.2.2.backport2/lib/rake/sprocketstask.rb:146:in `with_logger'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/gems/sprockets-rails-2.0.0.backport1/lib/sprockets/rails/task.rb:59:in `block (2 levels) in define'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/bin/ruby_executable_hooks:15:in `eval'
/Users/PikachuEXE/.rvm/gems/ruby-2.0.0-p247/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => assets:precompile
```

**Comment by clutchtutoring** [comment]
I would really appreciate any help I can get. I'm using the backport, here is my gemfile:
```
gem 'sprockets', github: 'tessi/sprockets', branch: '2_2_2_backport2'
gem 'sprockets-rails', '~> 2.0.0.backport1'
gem 'sass-rails', github: 'guilleiguaran/sass-rails', branch: 'backport'
```
For some reason, recently I updated my gemfile and this error started coming up every time I'd make a change to a CSS file:
`Warning. Error encountered while saving cache 9961881f387a45d76312eb0efe672ef47231dd21/admin.css.scssc: can't dump anonymous class #<Class:0x007fe4869df6a8>`

I have tried using different versions of the gems with no luck. This is slowing down development and also deployments. 

Any ideas on what I can try to make it work?
